src/frame-writer.cpp

```diff
@@ -279,9 +280,13 @@ void FrameWriter::init_codecs()
     }
 
     AVDictionary *dummy = NULL;
-    if (avformat_write_header(fmtCtx, &dummy) != 0)
+    char err[256];
+    int ret = avformat_write_header(fmtCtx, &dummy);
+    if (ret != 0)
     {
         std::cerr << "Failed to write file header" << std::endl;
+        av_strerror(ret, err, 256);
```

Comment:
same aas above, `averr()` should give you a string of the error.

Reply:
Same as above, averr gives garbage results.

Reply:
`ret` is not defined?

Reply:
@colemickens Nice catch, not sure what happened there. Fixed.